OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -191,6 +191,8 @@ IEnumerable<ActorInit> IActorPreviewInitInfo.ActorPreviewInits(ActorInfo ai, Act
 
 		bool IOccupySpaceInfo.SharesCell => false;
 
+		public bool CanLand => LandableTerrainTypes.Count > 0;
```

Comment:
This seems wrong. In Aircraft trait there's already a CanLand function that we use to determine wether we can land